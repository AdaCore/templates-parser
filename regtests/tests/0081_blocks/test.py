from test_support import *

run('testme', ["blocks2.tmplt"])
