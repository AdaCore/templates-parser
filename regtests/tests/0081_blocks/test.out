ligne1
  ---
  partie commune 1
  partie commune 2
    dans la table one
  partie commune 3
    dans la table "a value"
  ---
  partie commune 1
  partie commune 2
    in the table two
  partie commune 3
    in the table "a value"
  ---
  partie commune 1
  partie commune 2
    dans la table three
  partie commune 3
    dans la table "a value"
ligne2
